.github/workflows/example-ios-build-check.yml

```diff
@@ -45,43 +45,15 @@ jobs:
       - name: Check out Git repository
         uses: actions/checkout@v4
 
-      # TODO: Rework caching in this CI for monorepo setup.
```

Comment:
Let's keep such comment here instead:

`# TODO: Add caching for node_modules and artifacts that will work with monorepo setup.`